Pcap++/src/PcapLiveDevice.cpp

```diff
@@ -1,35 +1,35 @@
 #define LOG_MODULE PcapLogModuleLiveDevice

```

Comment:
@jafar75 this file contains many changes, maybe you ran a code formatter?

I'd suggest we limit this PR to adding `getIPv6Address()` and remove the other formatting changes. 
We have a task in our backlog to run code formatter on the entire code, we'll do it at some point...

Reply:
@jafar75 this file still has a lot of changes:

![image](https://user-images.githubusercontent.com/9059541/190954129-5c42d012-7bc9-4ff2-ac6f-f27652eae9a3.png)

I'd expect to only see the implementation of `getIPv6Address()`...